notebooks/multimodal_vdj/01_preprocessing.ipynb

```diff
@@ -0,0 +1,3169 @@
+{
```

Comment:
<ol><li>Please use markdown for the header levels. Jupyter book uses these for a table of content.</li><li>I really like the introduction to the biology.</li><li>The biology is there, but I think the motivation is kind of missing. WHY do want to know more about the receptors? What is this information useful for? Like what is the goal of the following analysis?</li><li>I feel like we should add some images there. We'll create one later with Biorender or so.</li><li>There are some TODOs. However, I feel like most of this is already covered in chapter https://github.com/theislab/extended-single-cell-best-practices/blob/development/jupyter-book/introduction/experimental_data_collection.md . Please don't introduce single-cell sequencing from scratch. Feel free to refer to the existing chapter and add the TCR/BCR sequencing relevant information, please.</li><li>Is "AIR" an abbreviation? What's the full name?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<ol><li>Done</li><li>Thanks</li><li>Done</li><li>Let's discuss Fri (27.05.22) what and how create</li><li>Good point, lets focus on VDJseq only</li><li>Added introduction of term AIR=Adaptive Immune Receptors = TCR+BCR, in contrast to IR, which includes more receptors.</li></ol>